lib/classes/Service.js

```diff
@@ -41,19 +41,33 @@ class Service {
       return BbPromise.resolve();
     }
 
+    let serverlessYamlPath = path.join(servicePath, 'serverless.yml');
```

Comment:
shouldn't this variable should be `serverlessYmlPath` to stay consistent with `.yml